unicodetools/src/main/java/org/unicode/tools/emoji/GenerateEmojiTestFile.java

```diff
@@ -271,7 +271,7 @@ public static void showLines(EmojiOrder emojiOrder, UnicodeSet charactersToInclu
                 for (String cp : vf.getCombinations()) {
                     charactersNotShown.remove(cp);
                     VariantStatus variantStatus = emojiOrder.emojiData.getVariantStatus(cp);
-		    String ageDisplay = "E" + BirthInfo.getVersionInfo(cp).getVersionString(2, 2);
+            String ageDisplay = "E" + BirthInfo.getVersionInfo(cp).getVersionString(2, 2);
```

Comment:
```suggestion
                    String ageDisplay = "E" + BirthInfo.getVersionInfo(cp).getVersionString(2, 2);
```

Reply:
Updated in https://github.com/unicode-org/unicodetools/pull/178/commits/7c5c10bdc0232617b772ea6dd0e1e9389a3a9cfc